package.json

```diff
@@ -189,7 +189,7 @@
 		"env:logs": "node ./tools/local-env/scripts/docker.js logs",
 		"env:pull": "node ./tools/local-env/scripts/docker.js pull",
 		"test:performance": "wp-scripts test-playwright --config tests/performance/playwright.config.js",
-		"test:php": "node ./tools/local-env/scripts/docker.js run -T php composer update -W && node ./tools/local-env/scripts/docker.js run php ./vendor/bin/phpunit",
```

Comment:
Moving these tasks works for me and shows a tiny improvement on my machine. That improvement will vary by machine and setup. Though tiny, worth doing 👍